src/robot/running/arguments/argumentspec.py

```diff
@@ -55,6 +55,10 @@ def minargs(self):
     def maxargs(self):
         return len(self.positional) if not self.var_positional else sys.maxsize
 
+    @property
+    def first_positional(self):
+        return self.positional and self.positional[0] or self.var_positional or None
```

Comment:
This property is somewhat confusingly named and doesn't feel too useful in general. Better to remove it and inline this code to the only place where it's used.

Reply:
Thanks, done!